MozillaReality/FirefoxReality/app/src/main/res/values/strings.xml

```diff
@@ -1146,6 +1324,26 @@
          when there are no exceptions. -->
     <string name="privacy_options_popups_list_empty_second">A list of sites with permissions will appear here.</string>
 
+    <!-- This string is displayed on top of the saved logins list in the Saved Logins panel. -->
+    <string name="privacy_options_saved_logins_list_header">Saved logins:</string>
+
+    <!-- This string is displayed in the first line of a text that's displayed in the middle of the saved logins panel instead of the logins list
+         when there are saved logins. -->
+    <string name="privacy_options_saved_logins_list_empty_first">When a login can be saved, you will see a dialog that let you save it.</string>
```

Comment:
Should it rather be "you will see a dialog that let**s** you save it"?

Reply:
Fixed.